src/schema-parser/asyncapi-schema-parser.ts

```diff
@@ -22,7 +94,8 @@ function getMimeTypes() {
     'application/schema+json;version=draft-07',
     'application/schema+yaml;version=draft-07',
   ];
-  ['2.0.0', '2.1.0', '2.2.0', '2.3.0'].forEach(version => {
+
+  specVersions.forEach((version: string) => {
```

Comment:
Removing the need for adding a new version every time a new spec version is out...